spec/chartmogul/customer_spec.rb

```diff
@@ -287,6 +287,32 @@
       expect(updated_customer.attributes[:custom][:meinung]).to eq ['lecker']
     end
 
+    it 'updates customer using class method', uses_api: true do
+      customer_uuid = 'cus_a29bbcb6-43ed-11e9-9bff-a3a747d175b1'
+
+      updated_customer = described_class.update!(customer_uuid, {
```

Comment:
If you add a field, does it break the VCR matching? I don't think it matches by body of POST request, so - this test might miss if we eg. started sending empty fields and deleting data if we somehow changed serialization.